src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    isMarried,
+    sex,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {(isMarried && sex === 'm') && (
```

Comment:
I guess it is better to create a function like this and do not hardcode
    const getPartner = () => {
      if (isMarried) {
        return `${partnerName} is my ${partner}`;
      }

      return 'I am not married';
  };